src/components/HomePage/TopBanner/TopBanner.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import Banner from 'components/Banner'
+import { t } from 'decentraland-dapps/dist/modules/translation/utils'
+
+import "./TopBanner.css"
+
+export default class TopBanner extends React.PureComponent {
+
+  render() {
+    return <Banner className="TopBanner" isClosable={true} name="dcl-top-banner">
+      <p>
+      <span>{t('banners.top_banner_left')}</span>
+      <a href="https://hack.decentraland.org/">{' '}{t('banners.top_banner_action')}{' '}</a>
```

Comment:
is this url correct? shouldn't it be `https://gamejam.decentraland.org` ?

Reply:
Yes, it is correct. Must be the same than docs.decentraland.org
<img width="551" alt="Screen Shot 2019-09-18 at 23 11 43" src="https://user-images.githubusercontent.com/208789/65207983-0bbf8480-da6a-11e9-8eae-b14b760db204.png">